index.html

```diff
@@ -19,116 +20,118 @@
+                    <div>
+                        <p class="text_after_images">Updating list of 50+ sources on distributed teams, remote
+                            work, and
+                            how to make it all work better</p>
+                    </div>
                 </div>
-            </div>
-            <div class="atribute">
-                <img class="second_page_photos" width="304px" height="176px" src="photos\5.jpg"
-                    alt="A few words about this blog platform">
-                <div>
-                    <p class="text_after_images">A few words about this blog platform, Ghost, and how this site was
-                        made</p>
+                <div class="atribute">
+                    <img class="second_page_photos" width="304" height="176" src="photos/5.jpg"
+                        alt="A few words about this blog platform">
+                    <div>
+                        <p class="text_after_images">A few words about this blog platform, Ghost, and how this site was
+                            made</p>
+                    </div>
                 </div>
-            </div>
-            <div class="atribute">
-                <img class="second_page_photos" width="304px" height="176px" src="photos\6.jpg"
-                    alt="modern remote working">
-                <div>
-                    <p class="text_after_images">How modern remote working tools get along with Old School
-                        Cowboy's methods
-                    </p>
+                <div class="atribute">
+                    <img class="second_page_photos" width="304" height="176" src="photos/6.jpg"
+                        alt="modern remote working">
+                    <div>
+                        <p class="text_after_images">How modern remote working tools get along with Old School
+                            Cowboy's methods
+                        </p>
+                    </div>
                 </div>
             </div>
-
-
         </section>
-
-    </artcile>
+    </article>
     <footer class="footer_black_square">
         <div class="square_first_text">
             <div class="bold_text scroll_text">DIGITAL PRODUCT DESIGN</div>
-            <div class="scroll_text" class="work">REMOTE WORK</div>
+            <div class="scroll_text work">REMOTE WORK</div>
             <div class="bold_text scroll_text">UX DESIGN</div>
-            <div class="scroll_text" class="teams">DISTRIBUTED TEAMS</div>
+            <div class="scroll_text teams">DISTRIBUTED TEAMS</div>
             <div class="bold_text scroll_text">CREATIVITY</div>
-            <div class="scroll_text" class="strategy">STRATEGY</div>
+            <div class="scroll_text strategy">STRATEGY</div>
             <div class="bold_text scroll_text">SUSPENSE</div>
-            <div class="scroll_text" class="growth">GROWTH</div>
+            <div class="scroll_text growth">GROWTH</div>
```

Comment:
why does `scroll_text` class go first on some lines and second on others. your code will look better if you're consistent with it. it is also easier to understand what happens here if `scroll_text` is always first